ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -96,32 +113,92 @@ const onNavigationStateChange = (
 class IdpLoginScreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-
+    const nowMs = new Date().getTime();
     this.state = {
-      requestState: pot.noneLoading
+      requestState: pot.noneLoading,
+      startingMs: nowMs,
+      deltaMs: nowMs
     };
   }
 
-  private handleOnError = (): void =>
+  private getSpidId = (): string =>
+    this.props.loggedOutWithIdpAuth
+      ? this.props.loggedOutWithIdpAuth.idp.entityID
+      : "idpID n/a";
+
+  public componentDidMount() {
+    this.props.dispatch(
+      idpLoginStart({
+        id: this.getSpidId(),
+        detail: "start"
```

Comment:
Can't see any reason to have an idpLoginStart action with detail of "start".

Reply:
also this action looks a duplicate of the `idpSelected` action, already tracked on MixPanel